frontend/src/Console.jsx

```diff
@@ -36,26 +47,70 @@ function Console(props) {
   }
 
   function toggleModal() {
-    
     setModal({ isOpen: !modalState.isOpen });
   }
 
+  function sendForm(formDetails) {
```

Comment:
Rename to `params` (also it should just be the params object, then construct the `FormData` here in this function)